hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/utils/db/DBStoreBuilder.java

```diff
@@ -174,21 +184,16 @@ private void processDBProfile() {
       dbProfile = this.configuration.getEnum(HDDS_DB_PROFILE,
           HDDS_DEFAULT_DB_PROFILE);
     }
+    LOG.debug("default profile:{}", dbProfile);
   }
 
   private void processTables() throws IOException {
-    if (tableNames.size() > 0) {
-      for (String name : tableNames) {
-        addTable(name, dbProfile.getColumnFamilyOptions());
-        LOG.info("Using default column profile:{} for Table:{}",
-            dbProfile.toString(), name);
-      }
+    List<String> list = new ArrayList<>(tableNames);
+    list.add(DEFAULT_COLUMN_FAMILY_NAME);
+    for (String name : list) {
+      LOG.debug("using default profile for table:{}", name);
+      addTableDefinition(name, dbProfile.getColumnFamilyOptions());
     }
-    addTable(StringUtils.bytes2String(RocksDB.DEFAULT_COLUMN_FAMILY),
```

Comment:
Can we pass the dbProfile's name to addTable so that we can LOG the profile name inside addTable?

Reply:
Thanks @xiaoyuyao for the review.

Both previously and with this patch only the default profile name is logged, no custom profile names.  (I guess the reason for that is that it's only a `ColumnFamilyOptions` object being passed to `addTable`, not a `DBProfile`, and only the latter has meaningful name).  The patch changes logging of the default profile to happen only once instead of for all tables using it, to avoid repetition.